main.cpp

```diff
@@ -75,378 +74,232 @@ send me a DM to check your pull request
+    3)is long session
+    4)is intro
+    5)location number
  */
 struct KaratePractice
 {
-    int timeSlot = 1;
-    int location =4;
-    bool isBoxing = false;
-
-    unsigned int numberOfStudents = 8;
-
-    struct StudentLesson
-    {
-     void takeLesson()
-     {
-
-     }
-     void posponeLesson()
-     {
-
-     }   
-    };
-
-    void practiceKarate(int timeSlot, int location, bool isBoxing);
-
-    StudentLesson karate, boxing;
+    
 };
-void KaratePractice::practiceKarate(int, int, bool)
-{
-    if(isBoxing == false)
-    {
-        karate.takeLesson();
-    }
-    else
-    {
-        boxing.posponeLesson();
-    }
-}
 /*
  5)
+3 things it can do:
+    1)start lesson
+    2)end lesson
+    3)pass student
+
+5 properties:
+    1)is car
+    2)is motorbike
+    3)is first lesson
+    4)is last lesson
+    5)is motorway special course
  */
 struct DrivingLesson
 {
-    bool hasKey = true;
-    int location = 1;
-
-    unsigned int numberOfStudents  = 7;
-
-    struct Exelerater
-    {
-       void drive()
-       {
-
-       } 
-       void reverse()
-       {
-
-       }
-    };
-
-    void takeLesson(int location, bool hasKey);
-
-    Exelerater fast, slow;
+    
 };
-void DrivingLesson::takeLesson(int, bool)
-{
-    if(hasKey == true)
-    {
-        fast.drive();
-    }
-    else
-    {
-        slow.reverse();
-    }
-}
+
 /*
  6)
+3 things it can do:
+    1)feed fish
+    2)clean fish
+    3)special full service
+
+5 properties:
+    1)is marine
+    2)is tropical
+    3)is pond
+    4)is coral
+    5)is special
  */
 struct FishCareRoutine
 {
-    int dayOfWeek = 2;
-    float timeOfDay = 12;
-
-    unsigned int fishTanks = 12;
-
-    struct FishDetails
-    {
-        void brush()
-        {
-
-        }
-        void fishFood()
-        {
-
-        }
-    };
-
-    void attendFish(int dayOfWeek);
-
-    FishDetails clean, feed;
+    
 };
-void FishCareRoutine::attendFish(int)
-{
-    if(dayOfWeek == 2)
-    {
-        clean.brush();
-    }
-    else
-    {
-        feed.fishFood();
-    }
-}
+
 /*
  7)
+3 things it can do:
+    1)prepare salad
+    2)make salad
+    3)serve salad
+
+5 properties:
+    1)is vegan
+    2)has nuts
+    3)contains meat
+    4)bowl size number
+    5)dressing type
  */
 struct SaladPrepList
 {
-    bool haveBowl = true;
-    bool haveSpoon = true;
-
-    int time  = 12;
-
-    struct Chef
-    { 
-
-        void stirFast()
-        {
-
-        }
-        void stirSlow()
-        {
-
-        }
-    };
-    void prepareSalad(int time, bool haveSpoon);
-
-    Chef spoon, whisk;
+   
 };
-void SaladPrepList::prepareSalad(int, bool)
-{
-    if(haveBowl == true)
-    {
-        spoon.stirFast();
-    }
-    else
-    {
-        whisk.stirSlow();
-    }
-}
+
 /*
  8)
+3 things it can do:
+    1)design website
+    2)price website
+    3)make website
+
+5 properties:
+    1)has pay service
+    2)has facebook
+    3)has video
+    4)has member login
+    5)is free
  */
 struct MakeWebsite
```

Comment:
Good, but clearly `MakeWebsite` is a horrible name for a class.  It's your #3 in the Things it can do.   
You're really describing 2 different things here: 
your actions describe what a `Web Designer` does. 
your Properties describe what a `Web Site` has.   
- [ ] you need to redo this one.